lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -823,15 +860,15 @@ mapping:
 
       instance_resource_url:
         type: str
-        default: https://galaxyproject.org/
         required: false
         desc: |
           URL of the support resource for the galaxy instance.  Used in activation
           emails.
 
+          Sample default 'https://galaxyproject.org/'
+
       blacklist_file:
         type: str
-        default: config/disposable_email_blacklist.conf
```

Comment:
I'd re-add this as "Example value" below.

Reply:
Done